pubtools/pulplib/_impl/client/client.py

```diff
@@ -181,10 +184,52 @@ def search_repository(self, criteria=None):
             response_f, lambda data: self._handle_page(Repository, search, data)
         )
 
+    def get_maintenance_report(self):
+        """Get maintenance report from maintenance repository
```

Comment:
Suggest a slight tweak to not expose implementation details:

```suggestion
        """Get the current maintenance mode status for this Pulp server.
```

i.e. no need to mention "maintenance repository".